restlicsharpclient/restliclient/request/Request.cs

```diff
@@ -45,6 +45,9 @@ public Request(ResourceMethod method, RecordTemplate input, Dictionary<string, L
             this.headers = headers;
             this.queryParams = queryParams;
             this.baseUrlTemplate = baseUrlTemplate;
+
+            AddHeader(RestConstants.kHeaderRestliProtocolVersion, RestConstants.kRestLiVersion20);
```

Comment:
This should be in your ClientUtil.buildRequest, no need to add in each Request class.